sdk/core/azure-core/inc/azure/core/version.hpp

```diff
@@ -22,7 +22,7 @@ namespace Azure { namespace Core {
   /**
    * @brief Provides version information.
    */
-  class Version {
+  class PackageVersion {
```

Comment:
Reminder to rename the header files too, later once all the namespace moves are done.

Reply:
Sure. But we will be doing all the file movements later, in separate PR(s). Please ping me if you expect to be fixed in this PR.

Reply:
Separate is good.

Reply:
There are lots of header renames we need to do.  We will do them after the types have final homes.